Neos.ContentRepository.Core/Classes/Projection/ContentGraph/NodePath.php

```diff
@@ -63,11 +72,6 @@ public function isRoot(): bool
         return $this->value === '/';
     }
 
-    public function isAbsolute(): bool
-    {
-        return strpos($this->value, '/') === 0;
-    }
-
```

Comment:
makes sense that this will be remove, commenting on it as it _was_ api. But useless api^^